MAUI/PDF-Viewer/Printing-PDF-files.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Printing PDF files in .NET MAUI PDF Viewer control | Syncfusion
+description: Learn here all about printing feature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and more.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Printing PDF Files in .NET MAUI PDF Viewer (SfPdfViewer)
+
+![Printing PDF Files in .NET MAUI PDF Viewer](Images\Print\print.png)
+
+![Print Preview Window](Images\Print\print-preview-window.png)
+
+
+Using [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html), you can effortlessly print PDF documents directly from your application code. This feature enables users to conveniently fill PDF form fields and print the document with the updated values incorporated.
+
+Moreover, when performing a print operation, the default system print dialog opens, providing users with familiar options and settings to customize their printing experience. Within this dialog, users can adjust settings such as printer selection, the number of copies, page orientation, and more, ensuring that their printed documents meet their specific requirements.
```

Comment:
Are all these operations available in all the platforms?

<img width="352" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/934f1650-db8c-48e5-a563-e4c16e994520">

Reply:
Yes, Printer selection, number of copies and page orientation available in all platforms.